css/style.css

```diff
@@ -15,5 +49,167 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.header, .footer {
+    padding: 0rem 1rem 0rem 1rem;
+    max-width: 1200px;
```

Comment:
setting a max width as this limit your design to fit screen size less or equal to this. Thia ia how it renders on larger screen, which isnt what you are trying to achieve. Please look into this, remove the max-width anywhere you have it and work on the structuring again
![Screenshot 2023-10-12 at 18 21 01](https://github.com/CodeYourFuture/HTML-CSS-Module-Project/assets/13808341/af27d85e-d6c9-4baf-97da-b1f857e51f61)

Reply:
I did the change